lib/subscribers/mcp-client.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2025 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const Subscriber = require('./base')
+const { MCP } = require('../metrics/names')
+const record = require('../metrics/recorders/generic')
+
+class McpClientSubscriber extends Subscriber {
+  constructor(agent) {
+    super(agent, '@modelcontextprotocol/sdk:nr_callTool')
+    this.events = ['asyncEnd']
+    this.requireActiveTx = true
+  }
+
+  handler(data, ctx) {
+    const toolName = data?.arguments?.[0]?.name
+    const name = `${MCP.TOOL}/callTool/${toolName}`
+    const segment = this._agent.tracer.createSegment({
+      name,
+      parent: ctx.segment,
+      recorder: record, // TODO: is the generic recorder sufficient?
+      transaction: ctx.transaction
+    })
+
+    const newCtx = ctx.enterSegment({ segment })
+    return newCtx
+  }
+}
+
+const mcpClientConfig = {
+  package: '@modelcontextprotocol/sdk',
+  instrumentations: [
+    {
+      channelName: 'nr_callTool',
+      module: {
+        name: '@modelcontextprotocol/sdk',
+        versionRange: '^1.13.0',
```

Comment:
if you do `^` caret matches it'll never match when 2.x is released. this should be `>=1.13.0`